understanding/22/dragging-movements.html

```diff
@@ -45,6 +45,8 @@ <h2>Intent of Dragging Movements</h2>
 
       <p>Some tasks may typically involve dragging, such as sorting items in a list, but as there are other ways to achieve that task they are not considered essential.</p>
       
+      <p>A special case are dragging movements that invoke animations keyed to the pointer position, for example, selection rectangles that set the first x/y rectangle coordinate at the pointer position via a pointer down-event, and the second x/y coordinate, after a dragging movement, at the next up-event. A similar example is would be a connecting line drawn between two different items on the screen, as in an allocation test where users are required to draw a line between questions and corresponding answers. In these cases, the dragging movement would require an alternative way to accomplish the same action that does not rely on the dragging movement. For example, two separate single tap or click actions may define the rectangle coordinates or the start and end points of a connecting line.</p>
```

Comment:
`A similar example is would be` > `A similar example would be`

Reply:
Done.